### DISCUSSION TITLE
App development - How to notify Github of Security Incident?

### DISCUSSION THREAD

**Discussion by friesenA** [community/community#33191] [Other Feature Feedback, Questions, & Ideas]
On Github's [Security Best Practices](https://docs.github.com/en/developers/github-marketplace/creating-apps-for-github-marketplace/security-best-practices-for-apps) it recommends that "You should have the capability to notify GitHub within 24 hours of a confirmed incident." but does not give any details.

In developing an incident response plan, contact details are vital to notify stakeholders. Through what secure mechanism would an app owner report a security compromise or incident to Github?

**Comment by radar-blip** [comment]
You can report such things here: [Support Page](https://support.github.com/request)
There is also a report link under each users profile picture.

**Reply by friesenA** [reply]
I'm afraid I cannot see any options in the Support Page which deal with reporting security incidents. Can you name the specific option you meant? When using the search functionality it only seems to turn up community posts which are not a private mechanism of contact.

As for the user reporting, I'm not looking to report any individual misbehaviour. I'm looking for a contact link to Github Security, so that in the occasion that an App my organization owns is compromised it can be reported as requested in their documentation.

**Reply by radar-blip** [reply]
I thought you might find something helpful there. I looked at it again and there isn't any particular way to contact Github Security in a case such as yours. (as you already know). I looked around a bit more but could not find anything useful.

You may have to ask Support or Staff directly what to do in your case. 
Sorry I could not be of any help.
